### ISSUE TITLE
nbdkit: error: invalid value "deflate" for boolean flag "--compress"

### ISSUE THREAD

**Issue by jeromerobert** [archiecobbs/s3backer#195]
The following invocation of `nbdkit` works:
```
nbdkit -f s3backer s3b_encrypt=true s3b_prefix=p/ s3b_accessEC2IAM=myiam s3b_no-vhost=true s3b_ssl=true s3b_region=eu-west-1 bucket=mybucket force=true s3b_compress=true
```

but changing  `s3b_compress` from `true` to `deflate` or `zstd` does not:

```
nbdkit: error: invalid value "deflate" for boolean flag "--compress"
```

```
nbdkit: error: invalid value "zstd" for boolean flag "--compress"
```

This is with version 2.0.2.

**Comment by archiecobbs** [comment]
Thanks - should be fixed by 6d5a276 (please confirm).

**Comment by jeromerobert** [comment]
Now `s3b_compress=zstd` or `s3b_compress=deflate` leads to `nbdkit: error: unknown s3backer parameter "compress"`. `s3b_compress=true` is still working.

**Comment by archiecobbs** [comment]
Oops, stupid bug - thanks for checking. Should be fixed by a7085ad.

**Comment by jeromerobert** [comment]
Ok now it works. Thank you.
